vllm/attention/backends/flash_attn.py

```diff
@@ -207,6 +209,8 @@ def forward(
                 )
             else:
                 # prefix-enabled attention
+                # TODO(Hai) this triton kernel has regression issue with
+                # FP8 KVCache to handle mixed types
```

Comment:
see same comment in xformers.py

Reply:
also do make sure to explain in code comment here

Reply:
solved